populate/src/test/java/bio/terra/pearl/populate/PopulateDemoTest.java

```diff
@@ -271,7 +271,7 @@ private void checkSurveyPopulation(UUID studyEnvironmentId) {
         List<StudyEnvironmentSurvey> socialHealthSurveys = studyEnvironmentSurveys.stream().filter((ses) -> ses.getSurvey().getStableId().equals("hd_hd_socialHealth")).toList();
         // check that the past versions of the social health survey are included
         assertThat(socialHealthSurveys, hasSize(3));
-        assertThat(socialHealthSurveys.stream().map(ses -> ses.getSurvey().getVersion()).toList(), contains(1, 2, 3));
+        assertThat(socialHealthSurveys.stream().map(ses -> ses.getSurvey().getVersion()).collect(Collectors.toSet()), contains(1, 2, 3));
```

Comment:
why was this necessary?  if the findSurveys is returning duplicates, we might have deeper problems

Reply:
I ran into a random test failure where it returned 3, 2, 1. It happened just once, though, 🤷